src/Commands/TerminusCommand.php

```diff
@@ -21,21 +24,34 @@
      * @var boolean True if the command requires the user to be logged in
      */
     protected $authorized = false;
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
 
     /**
      * TerminusCommand constructor
-     *
-     * @param Config $config Terminus configuration object
      */
-    public function __construct($config)
+    public function __construct()
     {
-        $this->config = $config;
+        $this->config = new Config();
+        $this->setLog(new ConsoleLogger($this->output()));
```

Comment:
That's a reasonable request!  ;)

Robo is designed to use league/container inflection to automatically inject your dependencies.  Just make your class instantiate a SomethingAwareInterface, and the inflector will call the appropriate set method provided by SomethingAwareTrait in the same class.

However, you should not be REQUIRED to use league/container to use the Robo runner.  I'll make a small change to the runner to allow you to pass in already-constructed command classes.